docs/yarn.lock

```diff
@@ -2681,7 +2681,7 @@ class-utils@^0.3.5:
     isobject "^3.0.0"
```

Comment:
Please revert.

Reply:
This has been reverted

Reply:
Seems like it's still here

Reply:
🤦 I'll fix it again

Reply:
This should be fixed again. 

Thank you for your patience as I work thru my first OSS contribution by the way!